ci/2.13/before_script.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd 2.13/mocks
```

Comment:
I‘m wondering if it's better to use relative path `cd ./mock`.

Reply:
The script will be run in the root folder of the project, so `2.13/mocks` means `./2.13/mocks`, that's [this](https://github.com/openservicebrokerapi/osb-checker/tree/master/2.13/mocks) folder.

Reply:
Got it